envs/monkey_zoo/terraform/monkey_zoo.tf

```diff
@@ -362,7 +372,7 @@ resource "google_compute_instance_from_template" "credentials-reuse-14" {
 
 resource "google_compute_instance_from_template" "credentials-reuse-15" {
   name = "${local.resource_prefix}credentials-reuse-15"
-  source_instance_template = local.default_linux
+  source_instance_template = local.default_ubuntu
```

Comment:
Where is this defined?

Reply:
<img width="578" alt="image" src="https://user-images.githubusercontent.com/15820737/196877570-26359a03-91cf-4ff0-bb6e-509eb0cf816f.png">

Reply:
ahh, so the  "linux" part was a typo?

Reply:
Yep.